Packs/Sixgill-Darkfeed/Integrations/Sixgill_DVE_Feed/Sixgill_DVE_Feed.py

```diff
@@ -0,0 +1,210 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+""" IMPORTS """
+from typing import Dict, Callable, Optional, Any
+from collections import OrderedDict
+import traceback
+import requests
+
+from sixgill.sixgill_request_classes.sixgill_auth_request import SixgillAuthRequest
+from sixgill.sixgill_feed_client import SixgillFeedClient
+from sixgill.sixgill_constants import FeedStream
+from sixgill.sixgill_utils import is_indicator
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+""" CONSTANTS """
+CHANNEL_CODE = "7698e8287dfde53dcd13082be750a85a"
```

Comment:
Are you OK with the fact that this channel code will be in a public github repo?

Reply:
yes its not a secet :)